sml-bin/redeploy-testlib.sml

```diff
@@ -0,0 +1,50 @@
+structure Path =  OS.Path
+structure File =  OS.FileSys
+
+fun printLn x = print (x ^ "\n")
+
+fun fileExists x = File.access (x,[])
+
+fun testLibName base slug =
+  Path.joinDirFile
+    {
+      dir = (Path.concat (base,slug)),
```

Comment:
Space after comma,after all,is helpful in reading,so they say.  I do not know why,in any instance,one would not use a space,every time,after a comma.

Reply:
I got used to it, because a lot of old SML doesn't use spaces in tuples.  Of course, a lot of old SML tries to immitate c-style function calls by excluding a space between uncurried function names and their product argument; this convention really annoys me, so I don't need to stick with the other convention for a partial tradition either.  I will change it if you all think it is better.

Reply:
It is the difference of my eyes seeing the comma, and not seeing it.  So white space in code, when not syntactically limited to white space breaking things when there can definitely benefit from having it.